app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, fname_to_copy, new_fname = command.split(" ")
+
+    if not command == "cp":
+        pass
+
+    if fname_to_copy == new_fname:
+        pass
```

Comment:
you can join this two if conditions with `and` and better raise an informative warning instead of pass